CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Version [3.3.1] - (2019-04-25)
```

Comment:
Usually we just put `(TBD)` as the date and change it when we actually do the release.